sdk/go/common/diag/colors/colors.go

```diff
@@ -226,7 +226,7 @@ func measureText(s string) int {
 	i := iterator{s}
 	var text, directive string
 	for i.next(&text, &directive) {
-		width += uniseg.GraphemeClusterCount(text)
+		width += uniseg.StringWidth(text)
```

Comment:
Width and grapheme cluster count have different meanings.
Was this a bug in measureText (because the variable is still called width)?

Reply:
For demonstration, "🇩🇪🏳️‍🌈" from their examples has cluster count 2, width 4.

https://go.dev/play/p/TXTBExVjxfb

Reply:
> Was this a bug in measureText (because the variable is still called width)?

Yes. Not sure that there was an equivalent in prior versions of the package. Nice to pick up a bug fix here, then.